app/src/main/res/values/strings.xml

```diff
@@ -69,8 +69,8 @@
     <string name="heat_date">Heat Date</string>
     <string name="bred_date"># Bred / Date</string>
     <string name="metalTagID">Metal Tag #</string>
-    <string name="initializing">Initializing…</string>
-    <string name="downloading_data">Downloading data…</string>
+    <string name="initializing">Checking for updates.</string>
```

Comment:
I'd rename the tag to `checking_for_updates` too but it's no big deal at this point, tbh